### DISCUSSION TITLE
ImportError: libarpack.so.2

### DISCUSSION THREAD

**Discussion by Ashutosh-432** [opencollab/arpack-ng#380] [Q&A]
Hi I am getting this error [ ImportError: libarpack.so.2 ].

could you pls help me , how can i install this file from source?

**Comment by fghoussen** [comment]
Not without such less informations! What's your distro? Error at build/run time? Use cmake/autotools? logs/traces ? etc

> how can i install this file from source?

https://github.com/opencollab/arpack-ng/blob/db55a7ff4180bb440a980d618e302eded0db52f3/README.md?plain=1#L104
